### ISSUE TITLE
Progress Bar While Importing Movies

### ISSUE THREAD

**Issue by qazero** [Radarr/Radarr#5579]
**Is your feature request related to a problem? Please describe.**
Right now there's no way to know the progress during large movie library imports or manual imports. Depending on the size of the library being imported this could take days.

**Describe the solution you'd like**
It would be nice to see a progress bar/percent showing when movies are being imported.

**Comment by SoulEater45** [comment]
This would be nice for people, who mount their media share over network. Hence, it takes a bit more time to process the files, and a progress bar would help to know how much it will take :)

**Comment by joelm3103** [comment]
Bumping in case it's lost. This would help immensely.

**Comment by Elschnuppero** [comment]
I agree. I'm transfering over network and i have no idea when its done. Corrupted a lot of movies because of early disrupt.

**Comment by hadim** [comment]
I would also love to have this feature!

**Comment by deuko** [comment]
Bump, same!

**Comment by bakerboy448** [comment]
A simple +1 on the OP would suffice

Comments that are not productive to the issue at hand such as +1, me too, i want this too, when will this be in, any updates, etc. are poor etitique and do not result in issues being worked on any faster. There are ~no active developers.

**Comment by IvanCharcos** [comment]
Any updates on this?

**Comment by bakerboy448** [comment]
The primary development team is engaged in over five applications, including Lidarr, Prowlarr, Radarr, Readarr, Whisparr, and Sonarr. Currently, there is approximately oneish active developer contributing to these projects, with none of the contributors working on them full-time. Additionally, all members involved, whether in development or support roles, are volunteers and do not receive compensation for their contributions. Each team member balances their commitments alongside full-time jobs, family responsibilities, and other personal obligations.

As a result, the list of tasks to be completed across these projects, along with the various backends and modules, is extensive, while the available time for addressing these tasks is limited.

clearly no one from the community believes this request to be a priority and thus no one has jumped in to help with it
